app/lib/cloneutils.js

```diff
@@ -46,7 +46,10 @@ export const clone = (repoURL, destination, progressCallback, token) => {
     }
 
     if (token) {
-      options.fetchOpts.callbacks.credentials = function () {
+      options.fetchOpts.callbacks.certificate_check = () => { // for libgit2 problem
```

Comment:
Configuration for nodegit to clone with access token, will shift to using electron bindings in the future for better performance